lib/utils/paste.js

```diff
@@ -0,0 +1,21 @@
+import {clipboard} from 'electron';
+
+const getPath = platform => {
+  switch (platform) {
+    case 'darwin': {
+      const filepath = clipboard.read('public.file-url');
```

Comment:
does this return an empty string if you have something else than a file URL in the clipboard? 🤔  
that is, will this code only intercept pasted file urls and nothing else? (not messing with pasted commands for example)

Reply:
Yeah, in my tests it returned an empty string. So an empty string would return out of the `processClipboard` method, where term would evaluate to false (no processing happened) and continue with paste. 

I could only see it messing with some non-file paste if some application writes to those specific formats during a copy?